tasks/cosign-verify-attestation.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: cosign verify attestation
+    tekton.dev/pipelines.minVersion: "0.19"
+    tekton.dev/tags: cosign, chains, attestation
+  labels:
+    app.kubernetes.io/version: "0.1"
+    operator.tekton.dev/provider-type: redhat
+  name: cosign-verify-attestation
+spec:
+  description: cosign-verify-attestation verifies the attestation of the image reference with the provided public key.
+  params:
+  - description: Public key used to verify the signature. Provided as a PEM-encoded string or as a valid cosign key reference.
+    name: PUBLIC_KEY
+    type: string
+  - description: Pull reference of the image to verify.
+    name: IMAGE
+    type: string
+  results:
+    - name: VERIFY_ATTESTATION_JSON
+      description: Result of verification in JSON format
+  steps:
+  - script: |
+      #!/usr/bin/env bash
+      echo "App Studio Utility Task $(context.task.name)"
+      /appstudio-utils/util-scripts/$(context.task.name).sh "$(params.IMAGE)" "$(params.PUBLIC_KEY)" "$(results.VERIFY_ATTESTATION_JSON.path)"
+    name: verify
+    image: appstudio-utils
+    volumeMounts:
+    - name: ca-certificates
+      mountPath: /etc/ssl/certs/
+  volumes:
+  - name: ca-certificates
+    secret:
+      secretName: chains-ca-cert
```

Comment:
Should this be customizable? Not sure if "chains" is the right name within this context.

Reply:
Yeah, would make sense to use an optional workspace here. That actually gives me an idea to refactor this to use `$SSL_CERT_DIR` provided by Tekton which includes the service CA certificate and the CA bundle.